core/cog-view.c

```diff
@@ -378,6 +397,33 @@ cog_view_set_use_key_bindings(CogView *self, gboolean enable)
     g_object_notify_by_pspec(G_OBJECT(self), s_properties[PROP_USE_KEY_BINDINGS]);
 }
 
+/**
+ * cog_view_set_viewport: (set-property viewport)
+ * @self: A view.
+ * @viewport: The viewport where the View belongs.
+ *
+ * Since: 0.20
+ */
+void
+cog_view_set_viewport(CogView *self, CogViewport *viewport)
```

Comment:
This will be a private function, which does not have the `COG_API` macro next to it, and we would add it to a `cog-view-private.h` header that is _not_ installed as an API header.

Reply:
By the way, we do not need to provide documentation to `gi-docgen` for private functions. Of course it's fine to have the documentation comment for developers.

Reply:
Ok but let's leave it as it is.